README.md

```diff
@@ -1,5 +1,20 @@
 # calens
 
-This program is a changelog generator that nicely prints a changelog file from
+`calens` is a changelog generator that nicely prints a changelog file from
 a directory with versioned subdirs containing files. A sample directory can be
 seen in [the restic repository](https://github.com/restic/restic/tree/master/changelog)
+
+# Installation
+
+To install and run calens locally, clone this repo and run from the command line `go install`.
```

Comment:
Please replace that paragraph with (replacing the double backticks with three):
```
To install and run calens locally, run the following commands:
``
git clone https://github.com/restic/calens.git 
cd calens
CGO_ENABLED=0 go install
``
```

The CGO option avoids the need to mess with system packages.

Reply:
done